### PR TITLE
Sprint 31 TLT-1976 SpeedGrader autosave warning for Firefox

### PR DESCRIPTION
**Pull request by douglashall**

### CODE REVIEW

**Comment by rascalking** [comment]
@rascalking

File: js/speedgrader.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$(document).ready(function(){
+    if ($.browser.mozilla && window.location.href.indexOf('gradebook/speed_grader')) {
```
Comment (rascalking) [inline js/speedgrader.js]: Shouldn't that be `(window.location.href.indexOf('gradebook/speed_grader') > -1)`?

Comment (douglashall) [inline js/speedgrader.js]: Yes it should.

**Comment by rascalking** [comment]
:+1:

**Pull request merged**